### DISCUSSION TITLE
Fantastic work - thank you. No issue, but a question

### DISCUSSION THREAD

**Discussion by mashoutsider** [upscayl/upscayl#375] [Q&A]
### Checklist

- [X] I have checked that this issue isn't addressed in [the Wiki](https://github.com/upscayl/upscayl/wiki).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Description

It's great that we can create custom models to use in the tool. I'm considering training my own model, to try to modify the effects of "General Photo - Fast Real ESRGAN". Are you able to direct me to a .pth file for this model?  For example, is this based on RealESRGAN_x4plus.pth? Thanks for sharing this great work

**Comment by NayamAmarshe** [comment]
The fast one is the v3 of Real-ESRGAN pth file found in the releases of the repository :)

**Comment by mashoutsider** [comment]
That’s great - thanks for the help! And good luck with the project
